pkg/common/webhook/webhook.go

```diff
@@ -166,7 +180,7 @@ func (w *Webhook) GetWebhookData(ctx context.Context, provider *Spec, ref *esv1b
 		return nil, fmt.Errorf("failed to create request: %w", err)
 	}
 	for hKey, hValueTpl := range provider.Headers {
-		hValue, err := ExecuteTemplateString(hValueTpl, data)
+		hValue, err := ExecuteTemplateString(hValueTpl, rawData)
```

Comment:
Isn't this a breaking change?

Reply:
It is, but I guess what's being sad here is that we don't want to use QueryEscape for Headers...